frontend/www/js/omegaup/components/RadioSwitch.vue

```diff
@@ -39,6 +39,7 @@ export default class RadioSwitch extends Vue {
   @Prop({ default: false }) valueForFalse!: any;
   @Prop({ default: T.wordsYes }) textForTrue!: string;
   @Prop({ default: T.wordsNo }) textForFalse!: string;
+  @Prop({ default: false }) value!: any;
 
   radioValue = this.selectedValue ?? false;
```

Comment:
ahora que lo pienso, esto no debería ser

```suggestion
  radioValue = this.selectedValue ?? this.valueForFalse;
```

Reply:
Cierto, no se había requerido, porque hasta el momento sólo se han utilizado valores boleanos